scripts/release/generate_history_notes.py

```diff
@@ -112,8 +113,7 @@ def get_commits():
         words = line.strip('"').split(None, 1)
         sha = words[0]
         subject = words[1]
-        if not subject.startswith('{'):
-            dev_commits.append({'sha': sha, 'subject': subject})
+        dev_commits.append({'sha': sha, 'subject': subject})
```

Comment:
Add dev commits even it starts with `{` as candidates to fetch corresponding PRs since we could make mistakes when committing it and would like to add history notes later.